modules/direct-io/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/IgniteNativeIoLib.java

```diff
@@ -150,10 +150,10 @@ else if (verIntComps.get(verIdx) == 2) {
 
             if (compsCnt > majorRevIdx && verIntComps.get(majorRevIdx) > 4)
                 return true;
-            else if (compsCnt > minorRevIdx
-                && verIntComps.get(majorRevIdx) == 4
-                && verIntComps.get(minorRevIdx) >= 10)
-                return true;
+            else
```

Comment:
It seems that braces are required here according to https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-BracesandIdentation

Reply:
Good cactch! Actually, this changes are not related to the PR.
So it has to be reverted. Thanks!